tests/Agent/IntegrationTests/Applications/OwinRemotingClient/Properties/PublishProfiles/LocalDeploy.pubxml

```diff
@@ -1,16 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm guessing we don't need the publish profile because this is a .NET Framework app?

Reply:
It was inadvertently added when I recreated the apps for StyleCop. Not needed because it's not an AspNet app.